api/kubernetes/cli/deploment.go

```diff
@@ -0,0 +1,22 @@
+package cli
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	labels "k8s.io/apimachinery/pkg/labels"
+)
+
+// @description Checks whether the given name is unique in the given namespace.
```

Comment:
```suggestion
// IsUniqueStackName Checks whether the given name is unique in the given namespace.
```

Reply:
maybe `HasStackName`? because isUnique is not right, as this functions says if the stack name is appearing as in the cluster (isUnique sounds like there's only one stack with this name)

Reply:
Done